features/hyphens.yml

```diff
@@ -2,3 +2,8 @@ name: Hyphenation
 description: The `hyphens` CSS property controls when long words are broken by line wrapping. Although called `hyphens`, the property applies to word-splitting behavior across languages, such as customary spelling changes or the use of other characters to mark an intraword line break.
```

Comment:
Can we mention in here somewhere that the rules used for breaking up words only work for English? This way, it makes more sense why there's also a non-english hyphens feature.